svn/trunk/inc/class-hyvor-talk-webpage.php

```diff
@@ -130,11 +130,14 @@ public function shortCodeComments($attr) {
 
 	public function shortCodeCount($attr) {
 		$this->setEmbedVariables(false);
+		ob_start();
 		include_once HYVOR_TALK_DIR_PATH . '/html/count.php';
+		$content = ob_get_clean();
 
-		$mode = isset($attr['mode']) && $attr['mode'] === 'number' ? "data-talk-mode=\"number\"" : '';
+		$pageId = isset($attr['id']) ? "page-id={$attr['id']}" : (is_single() ? "page-id=" . get_the_ID() : '');
```

Comment:
nesting shorthand if's isn't the best code. I suggest using if conditions here for clarity